### PR TITLE
Added longhorn multicluster

### PR DESCRIPTION
**Pull request by e-minguez**

### CODE REVIEW

File: fleet-examples/gitrepos/longhorn-gitrepo-multicluster.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+kind: GitRepo
+apiVersion: fleet.cattle.io/v1alpha1
+metadata:
+  name: longhorn
+  namespace: fleet-default
+spec:
+  repo: https://github.com/suse-edge/misc.git
+  branch: longhorn-fleet-default
```
Comment (atanasdinov) [inline fleet-examples/gitrepos/longhorn-gitrepo-multicluster.yaml:8]: Is this intentionally not `main`?

File: fleet-examples/gitrepos/longhorn-gitrepo-multicluster.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+kind: GitRepo
+apiVersion: fleet.cattle.io/v1alpha1
+metadata:
+  name: longhorn
+  namespace: fleet-default
```
Comment (atanasdinov) [inline fleet-examples/gitrepos/longhorn-gitrepo-multicluster.yaml:5]: The other GitRepos we have use `fleet-local`. Should we keep up the consistency?

File: fleet-examples/gitrepos/longhorn-gitrepo-multicluster.yaml (hunk 1 above)
Comment (e-minguez) [reply fleet-examples/gitrepos/longhorn-gitrepo-multicluster.yaml]: Good catch, thanks! I forgot to put it back to main :D

File: fleet-examples/gitrepos/longhorn-gitrepo-multicluster.yaml (hunk 2 above)
Comment (e-minguez) [reply fleet-examples/gitrepos/longhorn-gitrepo-multicluster.yaml:5]: fleet-local gitrepos are only applied to the local cluster (i.e.- where Rancher/Fleet is running). In order to be able to deploy things into clusters that are not local (i.e.- deployed or imported from Rancher) the assets need to live in the fleet-default namespace https://fleet.rancher.io/namespaces#gitrepo-namespace

**Review by atanasdinov — APPROVED** [review]

**Pull request merged**